docs/faq.md

```diff
@@ -67,15 +70,80 @@ Embed a special version of Semgrep Playground with an `iframe`. The source is `h
 <iframe title="Semgrep example no prints" src="https://semgrep.dev/embed/editor?snippet=ievans:print-to-logger" width="100%" height="432" frameborder="0"></iframe>
 ```
 
+## Usage limits
+
+This section describes usage limits.
+
+:::caution important dates
+* The Semgrep Community tier will be replaced with Team tier on May 23, 2023.
+* All users with any number of contributors are able to access Semgrep Supply Chain, Semgrep Pro Rules, and other paid offerings for free as of this update (May 15, 2023).
```

Comment:
CC @pabloest 

I agree with the concerns from Slack that this implies it's free. I wonder if we may be able to just remove bullets 2 and 3 for the actual FAQ below.

Reply:
Removed this!